CHANGELOG.md

```diff
@@ -1,8 +1,32 @@
 ## Change Log
 
-### v3.1.0-preview10 - 2020-06-09
+### v3.1.0-preview10 - 2020-08-12
 
+* Improve error handling during message parsing.
+* Add `MessageParseException` class.
 * Added `CauseOfDeathEntityAxisList` property to `CodingResponseMessage` and `CodingUpdateMessage`.
+* Updated `DeathRecord` class to match latest VRDR IG changes.
```

Comment:
Do we want to summarize a few of these, e.g. change of state from fully spelled out to state code?

Reply:
Will do.